ArgoProject/src/components/AI.h

```diff
@@ -0,0 +1,38 @@
+﻿#ifndef _COMPONENT_AI_H
+#define _COMPONENT_AI_H
+
+#include "commands/BaseCommand.h"
+
+namespace app::comp
+{
+	struct AI
+	{
+	public: // Constructors/Destructor/Assignments
+		AI() = default;
+		~AI() = default;
+
+		AI(AI const &) = default;
+		AI & operator=(AI const &) = default;
+
+		AI(AI &&) = default;
+		AI & operator=(AI &&) = default;
+
+	public: // Public Static Functions
+	public: // Public Member Functions
+	public: // Public Static Variables
+	public: // Public Member Variables
+		std::optional<app::Entity> m_currentNode;
```

Comment:
```suggestion
		std::optional<app::Entity> currentNode;
```
Component member variables don't have `m_`.